src/common.ts

```diff
@@ -21,6 +21,10 @@ export interface ObjectToStructConverterConfig {
   stringify?: boolean;
 }
 
+export interface Struct {
```

Comment:
I don't see this `Struct` being used, am I missing something?

Reply:
You were right!